kotlin-native/runtime/src/launcher/cpp/launcher.cpp

```diff
@@ -30,8 +30,12 @@ using namespace kotlin;
 //--- Setup args --------------------------------------------------------------//
 
 OBJ_GETTER(setupArgs, int argc, const char** argv) {
+  if (argc > 0 && argv[0][0] != '\0') {
```

Comment:
Need a comment explaining the purpose of `argv[0][0] != '\0'` check.

Reply:
You're right, I also commented it here and not only indirectly in the testcase.